src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1073,20 +1073,15 @@ public function getNode(int $position)
         return $this->nodes[$position] ?? null;
     }
 
-    /**
-     * @return int
-     */
-    #[\ReturnTypeWillChange]
-    public function count()
+    public function count(): int
     {
         return \count($this->nodes);
     }
 
     /**
-     * @return \ArrayIterator|\DOMNode[]
+     * @return \ArrayIterator<\DOMNode>
```

Comment:
```suggestion
     * @return \ArrayIterator<int, \DOMNode>
```

Reply:
Yeah, I know. As I said, I'm going to fix all those issues later.

Reply:
(later as in _before_ we merge, of course)